server/src/main/resources/countries/countries_DE.csv

```diff
@@ -0,0 +1,249 @@
+AF,Afghanistan
```

Comment:
Das `countries` Verzeichnis könnte auch in das `enum-data` Verzeichnis verlegt werden.